File: fibonacci.cpp
#include<bits/stdc++.h>
#include<stdio.h>
using namespace std;

int fib(int n)
{
    if (n <= 1)
        return n;
    return fib(n-1) + fib(n-2);
}

int main ()
{
    int n;
    printf("Enter the Fibonacci Number which you wish to see in the Fibonacci Series: ");
    scanf("%d", &n);
    cout << fib(n);
    getchar();
    return 0;
}

File: fibonacci.java
public class fibonacci
{  
    public static void main(String args[])  
    {    
    int n1=0;
    int n2=1;
    int n3;
    int i,t=25;    
    System.out.print(n1+" "+n2);
    for(i=2;i<t;++i)
    {    
          n3=n1+n2;    
          System.out.print(" "+n3);    
          n1=n2;    
          n2=n3;    
    }    
      
    }
}  